src/nginxconfig/templates/global_sections/index.js

```diff
@@ -33,3 +33,4 @@ export { default as Performance } from './performance';
 export { default as Logging } from './logging';
 export { default as NGINX } from './nginx';
 export { default as Tools } from './tools';
+export { default as Docker } from './docker';
```

Comment:
```suggestion
export { default as Docker } from './docker';
export { default as Tools } from './tools';
```
Tools is a bit of a tool-meta tab, so we should put config generation tabs before it :)